added the files from Metalab API

cerberlink opened the pull request:
**DESCRIPTION**
I added the 4 filenames for API. 

**TESTING**
I did not test anything.

cerberlink committed:
added the files from Metalab API

ekno27 reviewed (changes requested):
Few things: 
1. Function still needs to be created to parse through the data received from the API, this function can be implemented by using the data in `filter+courses.js` as for the other files that contain data, those will probably not be used for the application
2. For future reference, the heading should look like: 
CLS--005--DEV: Filter API results, so we  know what task this is

cerberlink committed:
fixed the covert function and added the end_time, too

cerberlink committed:
CLS-005-DEV Add function for end_time.

cerberlink commented:
DESCRIPTION
I added the end_time and put the loop as the comment. 

TESTING
Before I tested them successfully, today I modified the start_time with [''] and removed the time with [0] because it cannot read property '0' of undefined. When I fixed them, the program ran well. However, the output displays undefined:NaN A.M. I believe something in bug is on 62-68.

ekno27 committed:
merge dev
